persistence/ravendb/cluster-configuration.md

```diff
@@ -7,28 +7,16 @@ versions: '[6.0,)'
+Using cluster-wide transactions correctly is complex. Cluster-wide transactions make use of compare-exchange values in order to guarantee consistent writes across the majority of nodes in the cluster. With the current RavenDB api, it’s up to the user to properly implement that across the business data and the NServiceBus data.
 
-Using cluster-wide transactions correctly requires all the following:
-- Modify all existing documents to store a synchronization identifier. This can be part of the document, or can be stored inside the document’s metadata.
-- Create a corresponding compare-exchange value for every document. The value of the compare-exchange value must be set to the synchronization identifier’s value.
-- When updating any document, the code must:
-  - Load the document and its metadata
-  - Load the corresponding compare-exchange value
-  - Compare the document’s synchronization identifier with the one stored in the compare-exchange value that corresponds to the document. If they are not the same, it means that the document has already been changed and both documents must be reloaded.
-  - Update the document as well as the synchronization identifier
-  - Update the compare-exchange value’s document value with the new synchronization identifier
-
-Given the complexity involved in implementing cluster-wide transactions properly across the business data and the NServiceBus data, Particular no longer supports clusters in RavenDB. NServiceBus.RavenDB can use cluster-wide transactions and compare-exchange values to correctly and safely store NServiceBus data, but cannot ensure any business data modified in an NServiceBus handler is safe. Each handler would need to be reassessed using the steps listed above to ensure correct usage of cluster-wide transactions.
-
-Not using cluster-wide transactions in the correct form for the business data could result in data loss that is impossible to detect. Due to the potential for data loss, support for RavenDB clusters was withdrawn. Unless cluster-wide transactions were already used in the correct form, introducing them comes with a very severe impact on the system.
+DANGER: Not using cluster-wide transactions in the correct form for the business data could result in data loss that is impossible to detect. Due to the potential for data loss, support for RavenDB clusters was withdrawn.
 
 ## How to move forward
 
-- If the RavenDB server is configured in a cluster, move back to using a single node. This is not ideal, as it introduces a single point of failure in the system.
-- Move to another persistence option. This would require migrating data from RavenDB to [something else](/persistence).
+- If the RavenDB server is configured in a cluster, move back to using a single node. This may not be ideal, as it introduces a single point of failure in the system.
```

Comment:
```suggestion
- If the RavenDB server is configured in a cluster, move back to using a single database node. This may not be ideal, as it introduces a single point of failure in the system.
```

Reply:
This is included in Tim's suggestion which I've incorporated.